lib/components/base_component.dart

```diff
@@ -23,7 +24,13 @@ abstract class BaseComponent extends Component {
   final OrderedSet<Component> _children =
       OrderedSet(Comparing.on((c) => c.priority));
 
-  List<Component> get children => _children.toList(growable: false);
+  /// The children list shouldn't be modified directly, that is why an
+  /// [UnmodifiableListView] is used. If you want to add children use the
+  /// addChild method, and if you want to propagate something to the children
+  /// use the propagateToChildren method.
```

Comment:
Wrap the propagateToChildren with [], so it will create a link on the docs

Reply:
Fixed!